src/main/fc/runtime_config.c

```diff
@@ -15,10 +15,16 @@
  * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+// FIXME this file has started to resemble a garbage bin.  move everything out into appropriate locations.
+
 #include <stdbool.h>
 #include <stdint.h>
 
+#include "config/parameter_group.h"
+#include "config/parameter_group_ids.h"
```

Comment:
no, in CF, include files dont include other include files - this was done so that it's obvious where the dependency issues are so that they don't get out of hand.